### PR TITLE
Add running of specs against rails 5

### PR DESCRIPTION
**Pull request by vipulnsward**
All seems good here.

### CODE REVIEW

**Comment by rmosolgo** [comment]
Awesome, thanks for getting this started!

```
Gem::InstallError: activesupport requires Ruby version >= 2.2.2.
An error occurred while installing activesupport (5.0.0.beta1), and Bundler
```

Looks like we'll have to only run Rails 5 tests with 2.2+.  (And .... I don't know how to do that!)

**Comment by vipulnsward** [comment]
:-) Let me look at that.

**Comment by vipulnsward** [comment]
@rmosolgo All's good now.

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
+  - 2.2.4
   - 2.1
-  - 2.0.0
-  - 1.9.3
-  - jruby-19mode
-before_script: 'bundle exec appraisal install'
-script: 'bundle exec rake appraisal test'
+  - 2.0
+  - 1.9
+  - jruby-9.0.1.0
```
Comment (rmosolgo) [inline .travis.yml]: Are we not supporting jruby-19mode anymore?

Comment (vipulnsward) [inline .travis.yml]: This is 1.9 mode, and hence fails for Rails 5 right now(Specified this to pick the latest JRuby)
2.2 mode is not out yet on travis..

Comment (rmosolgo) [inline .travis.yml]: Oh, I see, thanks!

> Ruby 1.9 mode is default in JRuby 1.7.0 and later, 1.8.7 compatibility can still be used with the --1.8 flag.

https://github.com/jruby/jruby/wiki/FAQs#does-jruby-support-ruby-19x-features

**Comment by rmosolgo** [comment]
:tada: !

**Pull request merged**